portfolio/src/main/webapp/index.html

```diff
@@ -55,6 +55,17 @@ <h4>Feeling random today? Click here to get a random message:</h4>
         <button onclick="addRandomGreeting()">Hello</button>
         <div id="greeting-container"></div>
       </div>
+      <div id="comments-section">
+        <form action="/comments" method="POST">
+          <p>Enter any comments (multiple comments are separated by enter):</p>
+          <textarea name="text-input" placeholder="Enter anything you like~" rows="5" cols="50"></textarea>
+          <br/><br/>
+          <input type="submit" />
+        </form>
+        <br>
+        <button onclick="getCommentsContent()">Fetch comments</button>
```

Comment:
consider doing this on page load in your `script.js` file using `document.addEventListener("DOMContentLoaded", function(){
  getCommentsContent();
});`

Reply:
Thanks! this is a much better approach! :)